rs/sns/governance/src/sns_upgrade.rs

```diff
@@ -640,20 +640,20 @@ pub struct GetProposalIdThatAddedWasmResponse {
     pub proposal_id: ::core::option::Option<u64>,
 }
 
-#[derive(Clone, PartialEq, candid::CandidType, candid::Deserialize)]
-pub(crate) struct ListUpgradeStepsRequest {
+#[derive(Clone, PartialEq, candid::CandidType, candid::Deserialize, Debug)]
+pub struct ListUpgradeStepsRequest {
```

Comment:
I don't get why we need this type in SNS Governance. Isn't this an endpoint of SNS-W?

Reply:
Does this just duplicate the types from SNS-W?